apps/web/src/pages/Bookings/Bookings.tsx

```diff
@@ -1,12 +1,82 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useQuery } from '@tanstack/react-query';
 import { bookingQueryKeys } from '@/services/bookings/request';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import styles from './Bookings.module.css';
+import { useDisclosure } from '@mantine/hooks';
+import { Drawer } from '@mantine/core';
+import { Booking } from '@/models/booking';
+
+interface event {
+  timeText: string;
+  event: {
+    title: string;
+    extendedProps: {
+      booking: Booking;
+    };
+  };
+}
+
 function Bookings() {
-  const { data } = useQuery({ ...bookingQueryKeys.list() });
+  const { data: bookings } = useQuery({ ...bookingQueryKeys.list() });
+  const [opened, { open, close }] = useDisclosure(false);
+  const [selectedBooking, setSelectedBooking] = useState<Booking | null>(null);
+
+  const mappedBookings = bookings?.map((booking) => ({
+    id: booking.id,
+    title: booking.title,
+    start: booking.startDate,
+    end: booking.endDate,
+    extendedProps: { booking },
+  }));
+
+  const handleEventClick = (booking: Booking) => {
+    setSelectedBooking(booking);
+    open();
+  };
+
+  const renderEventWithOpen = (eventInfo: event) => {
+    const booking = eventInfo.event.extendedProps.booking;
+    return renderEventContent(eventInfo, () => handleEventClick(booking));
```

Comment:
puedes dejarlo asi yo creo, no creo que agregemos mas cosas aqui
```suggestion
   
  const renderEventWithOpen = (eventInfo: event) => {
    const booking = eventInfo.event.extendedProps.booking;
    return (
      <div
        onClick={() => handleEventClick(booking)}
        className={styles.bookingDescription}
      >
        <b>{eventInfo.timeText}</b> <i>{eventInfo.event.title}</i>
      </div>
    );
  };

```